README.adoc

```diff
@@ -484,3 +479,21 @@ NOTE: If I can figure out a good way to setup a Ruby testing environment I'll do
     ...
 </configuration>
 -----
+
+=== Add version and build date to the header
+
+Automatically add version details to header and footer to all documents.
+
+[source, xml]
+-----
+<configuration>
+    ...
+    <attributes>
+        ...
+        <revnumber>${project.version}</revnumber>
+        <revdate>${maven.build.date}</revdate>
```

Comment:
I'd rather keep the tip simple and void other plugins integration.
Instead I fixed the use of `date` by `timestamp` property and also added how to deifne the date format.